docs/ml-clustering.md

```diff
@@ -265,3 +265,44 @@ Refer to the [R API docs](api/R/spark.gaussianMixture.html) for more details.
 </div>
 
 </div>
+
+## Power Iteration Clustering (PIC)
+
+Power Iteration Clustering (PIC) is  a scalable graph clustering algorithm
+developed by <a href=http://www.cs.cmu.edu/~frank/papers/icml2010-pic-final.pdf>Lin and Cohen</a>.
```

Comment:
It seems that `<a>` tag doesn't work here. Maybe, could you check the generated document and try `[Lin and Cohen](http://www.cs.cmu.edu/~frank/papers/icml2010-pic-final.pdf)` instead?

Reply:
I normally check the md file on the github. The link works OK. Is there a better way to check? @dongjoon-hyun @felixcheung 
https://github.com/apache/spark/blob/9158da8cb76cc13f3011deaa7ac2c290eef62389/docs/ml-clustering.md
I guess I will still remove the ```a href=``` since no other places in the doc uses ```<a>```

Reply:
Actually, I built this PR on my Mac, and found that the hyperlink is not generated.

Reply:
Thanks. I will change the hyperlink.